src/main/java/io/github/townyadvanced/townyprovinces/TownyProvinces.java

```diff
@@ -1,213 +1,238 @@
+			}
+			if (!MapDisplayTaskController.isMapSupported()) {
+				info("Did not find a supported map plugin. Cannot enable map integration.");
+				return false;
+			}
+			MapDisplayTaskController.startTask();
+			return true;
+		} catch (Exception e) {
+			Messaging.sendErrorMsg(Bukkit.getConsoleSender(), "Problem enabling map integration: " + e.getMessage());
+			e.printStackTrace();
+			return false;
+		}
+	}
+	
+	private boolean checkTownyVersion() {
+		if (!townyVersionCheck()) {
+			severe("Towny version does not meet required minimum version: " + requiredTownyVersion.toString());
+			return false;
+		} else {
+			info("Towny version " + getTownyVersion() + " found.");
+			return true;
+		}
+	}
+	
+	private boolean loadConfig() {
+		try {
+			Settings.loadConfig();
+		} catch (TownyInitException e) {
+			e.printStackTrace();
+			severe("Config.yml failed to load! Disabling!");
+			return false;
+		}
+		info("Config.yml loaded successfully.");
+		return true;
+	}
+
+	public static boolean loadLocalization(boolean reload) {
+		try {
+			Plugin plugin = getPlugin(); 
+			Path langFolderPath = Paths.get(plugin.getDataFolder().getPath()).resolve("lang");
+			TranslationLoader loader = new TranslationLoader(langFolderPath, plugin, TownyProvinces.class);
+			loader.load();
+			TownyAPI.getInstance().addTranslations(plugin, loader.getTranslations());
+		} catch (TownyInitException e) {
+			e.printStackTrace();
+			severe("Locale files failed to load! Disabling!");
+			return false;
+		}
+		if (reload) {
+			info(Translatable.of("msg_reloaded_lang").defaultLocale());
+		}
+		return true;
+	}
+
+	private boolean validateWorld() {
+		info("Now validating world");
+		World world = TownyProvincesSettings.getWorld();
+		if(world != null) {
+			info("World Validated");
+			return true;
+		} else {
+			Messaging.sendErrorMsg(Bukkit.getConsoleSender(), Translatable.of("msg_err_unknown_world"));
+			return false;
+		}
+	}
+	
+	private boolean registerListeners() {
+		PluginManager pluginManager = this.getServer().getPluginManager();
+		pluginManager.registerEvents(new TownyListener(), this);
+		return true;
+	}
+	public String getVersion() {
+		return this.getDescription().getVersion();
+	}
+
+	public static TownyProvinces getPlugin() {
+		return plugin;
+	}
+	
+	/**
+	 * Use this in most cases, as it looks better
+	 * @return plugin prefix
+	 */
+	public static String getTranslatedPrefix() {
+		return Translatable.of("townyprovinces_plugin_prefix").translate(Locale.ROOT);
+	}
+	
+	private boolean townyVersionCheck() {
+		return Version.fromString(getTownyVersion()).compareTo(requiredTownyVersion) >= 0;
+	}
+
+	private String getTownyVersion() {
+		return Bukkit.getPluginManager().getPlugin("Towny").getDescription().getVersion();
+	}
+
+	public static void info(String message) {
+		plugin.getLogger().info(message);
+	}
+
+	public static void severe(String message) {
+		plugin.getLogger().severe(message);
+	}
+}

```

Comment:
🚧 
- In this PR, something weird has happened to the TownyProvinces and ConfigNodes files
- As you can see, the pull request shows that almost everything was changed in those files, but obviously that is not what happened.
- It's probably something to do with automatic formatting in your IDE.
- Please take a look.